website/docs: stages/mtls: Clean up stage configuration section (#15753)
goauthentik/authentik: website/docs/add-secure-apps/flows-stages/stages/mtls/index.md

1. Log in as an admin to authentik, and go to the Admin interface.

> 2. From the authentik Admin inteface, navigate to **System** > **Certificates**, and either generate or add the certificate you’ll use as a certificate authority.

Author: oops

> 3. Then, from the authentik Admin interface, navigate to **Flows and Stages** > **Stages** and click **Create**. Select **Mutual TLS Stage**, click **Next**, and define the following fields:

Reviewer: ```suggestion
3. Then, navigate to **Flows and Stages** > **Stages** and click **Create**. Select **Mutual TLS Stage**, click **Next**, and set the following fields:
```

>     - **Name**: define a descriptive name, such as "chrome-device-trust".

Author: ```suggestion
    - **Name**: provide a descriptive name, such as "chrome-device-trust".
```

>     - **Stage-specific settings**

Reviewer: ```suggestion
    - **Stage-specific settings**:
```
